ivy-test.el

```diff
@@ -241,6 +241,36 @@ will bring the behavior in line with the newer Emacsen."
                   (ivy--regex "(foo bar"))
                  "(\\(foo).*?(bar)")))
 
+(defmacro ivy--string-buffer (text body)
+  "Test helper that wraps TEXT in a temp buffer while running BODY."
+  `(with-temp-buffer
+    (insert ,text)
+    (goto-char (point-min))
+    ,body))
```

Comment:
It is more common and general to define `body` as a [`&rest` argument](https://www.gnu.org/software/emacs/manual/html_node/elisp/Argument-List.html) and then splice it into the quoted form with `,@body`.

Is moving to `(point-min)` necessary?

Reply:
Thanks, I'll adjust it. The test passes without the `goto-char` call so I guess it isn't. I'd thought the point would be outside of the word after insert, but apparently not so.

Thanks again for the detailed review.